### PR TITLE
Building Atoms

### PR DESCRIPTION
**Pull request by AymanMorsy**
## Description
-  Building [Buttons + sliders +form] Atom
- Moving Atoms from gallery.html to index.html

### CODE REVIEW

**Review by eslamheshamm — COMMENTED** [review]
Good Work Aymen,

- Be more descriptive in the title
- Buttons sizes look good but the font-weight need to adjust
-  Try to write more styles by tailwind classes inside the class attribute, not in a separate file

**Comment by eslamheshamm** [comment]
And the description should be shorter and contain the main changes you've done

**Comment by AymanMorsy** [comment]
hi @eslamheshamm , 
> 
> * Try to write more styles by tailwind classes inside the class attribute, not in a separate file

can you explain this point a little plz?

File: static/public/index.html [hunk 1]
```diff
@@ -3,12 +3,2201 @@
...
   <body>
+    <!-- ✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅ Ayman Morsy End  ✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅✅-->
+
+    <p class="bg-primary-shd6 text-center">Start of Ayman Morsy Atoms</p>
+     <section class="buttons-atoms">
+    <h1 class="text-4xl font-bold">Buttons</h1>
+    <h2 class="bg-grey-shd5">Primary</h2>
+    <div>
+      <div class="m-3">
+        <button class="btn-primary-large hover:bg-primary focus:bg-primary-shd1 focus:border-2 focus:border-primary">Button Text</button>
```
Comment (ayaosama05) [inline static/public/index.html:23]: Review Focus on button You need to cancel the default outline

File: static/public/index.html [hunk 2]
```diff
@@ -3,12 +3,2201 @@
...
+          <path
+            d="M10.293 0.292969L5.99997 4.58597L1.70697 0.292969L0.292969 1.70697L5.99997 7.41397L11.707 1.70697L10.293 0.292969Z" />
+        </svg>
+      </button>
+    </div>
+
+    <h2 class="bg-grey-shd5">Text</h2>
+    <div>
+      <div class="m-3">
+        <button class="txt-btn-large hover:text-accent focus:text-accent ">Button Text</button>
```
Comment (ayaosama05) [inline static/public/index.html]: For text buttons Remove the default outline and use underline as same as design

File: static/public/index.html [hunk 3]
```diff
@@ -3,12 +3,2201 @@
...
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M0.999801 7.66669H1.66647V9.00002V12.3334C1.66647 13.0687 2.26447 13.6667 2.9998 13.6667H4.9998H8.9998H10.9998C11.7351 13.6667 12.3331 13.0687 12.3331 12.3334V9.00002V7.66669H12.9998C13.2691 7.66669 13.5131 7.50402 13.6158 7.25536C13.7191 7.00602 13.6618 6.71936 13.4711 6.52869L7.47113 0.528691C7.21047 0.268024 6.78913 0.268024 6.52847 0.528691L0.528467 6.52869C0.337801 6.71936 0.280467 7.00602 0.383801 7.25536C0.486467 7.50402 0.730467 7.66669 0.999801 7.66669ZM5.66647 12.3334V9.00002H8.33313V12.3334H5.66647ZM6.9998 1.94269L10.9998 5.94269V9.00002L11.0005 12.3334H9.66647V9.00002C9.66647 8.26469 9.06847 7.66669 8.33313 7.66669H5.66647C4.93113 7.66669 4.33313 8.26469 4.33313 9.00002V12.3334H2.9998V9.00002V7.00002V5.94269L6.9998 1.94269Z"
+              fill="#94ACB5"
+            />
+          </svg>
+        </div>
+      </div>
+      <div>
```
Comment (ayaosama05) [inline static/public/index.html:640]: Focus here occurs on both of them separately but we need to show focus effect include prefix + input text together

**Comment by ayaosama05** [comment]
Good work Ayman 
- remove default outline for all buttons in focus mode
- Make focus as same as design like underline in text buttons
- In forms text input we need to show focus effect for all input include prefix and text input

**Comment by ayaosama05** [comment]
Secondary button in hover have a different background grey-bg not white and with focus have a border with color #5034A6

**Review by ayaosama05 — APPROVED** [review]
Good work

**Pull request merged**